Completed the Read Me File

nensley opened the pull request:
Read Me File answers required project questions, including a data description, algorithm description, tools used, and ethical concerns.

nensley committed:
Completed the Read Me File with details on the Application

Hy-Koo committed:
Merge branch 'main' into nathan-ensley

Hy-Koo reviewed (changes requested):
In the first sentence, 'The purpose of the project is to show the predicted income based on the input variables like age, sex, ...' would be better.

nensley committed:
Update README.md

Updated the Abstract to better align with application goal of predicting income.
